File: Section Reports/Preview/Hyperlinks and DrillThrough/C#/Hyperlinks and DrillThrough/DrillThrough2.cs
namespace GrapeCity.ActiveReports.Samples.HyperlinksAndDrillThrough
{
	public partial class DrillThrough2 : GrapeCity.ActiveReports.SectionReport
	{
		public DrillThrough2()
		{
			InitializeComponent();
			//Connect to Sample Database
			//
			Data.OleDBDataSource dS = new Data.OleDBDataSource();
			dS.ConnectionString = Properties.Resources.ConnectionString;
			dS.SQL = "Select * from [order details] where orderID = <%orderID||1%> order by productid";
			DataSource = dS;
		}
	}
}

File: Section Reports/Preview/Hyperlinks and DrillThrough/VB.NET/Hyperlinks and DrillThrough/DrillThrough2.vb
Public Class DrillThrough2
	Inherits GrapeCity.ActiveReports.SectionReport
	Public Sub New()
		MyBase.New()
		InitializeComponent()
		'Connect to Sample Database
		'
		DataSource = New Data.OleDBDataSource()
		CType(DataSource, Data.OleDBDataSource).ConnectionString = My.Resources.ConnectionString
		
		CType(DataSource, Data.OleDBDataSource).SQL = "Select * from [order details] where orderID = <%param:orderID||1%> order by productid"
		Dim paramOrderID As New SectionReportModel.Parameter()
		paramOrderID.Key = "orderID"
	End Sub
	Public ds As GrapeCity.ActiveReports.Data.OleDBDataSource
End Class
